src/openforms/js/components/form/i18n.js

```diff
@@ -0,0 +1,29 @@
+import * as en from '../../lang/formio/en.json';
+import * as nl from '../../lang/formio/nl.json';
```

Comment:
let's use the 'absolute' imports here, webpack is configured appropriately :)

```suggestion
import * as en from 'lang/formio/en.json';
import * as nl from 'lang/formio/nl.json';
```